oshi-core/src/main/java/oshi/software/os/OSThread.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * MIT License
+ *
+ * Copyright (c) 2010 - 2020 The OSHI Project Contributors: https://github.com/oshi/oshi/graphs/contributors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package oshi.software.os;
+
+/**
+ * Represents a Thread/Task on the operating system.
+ */
+public interface OSThread extends OSProcess {
+
+    /**
+     * <p>
+     * Getter for the field <code>threadId</code>.
+     * </p>
+     *
+     * @return Returns the id of the thread.
+     */
+    int getThreadId();
+
+    /**
+     * <p>
+     * Getter for the field <code>parentProcess</code> which is the parent process of this thread.
```

Comment:
Let's be very clear here that this is the parent of the process to which the thread belongs. You might also suggest that the thread's *owning* process is accessible with getProcessId().

Reply:
Yes true, i was thinking about this but then i let it be like that. Will change this.